pkgs/by-name/tu/tuxguitar/package.nix

```
@@ -36,7 +43,7 @@ stdenv.mkDerivation (finalAttrs: {
     ln -s $out/share $out/bin/share
 
     wrapProgram $out/bin/tuxguitar \
-      --set JAVA "${jre}/bin/java" \
+      --set PATH "$PATH:${jre}/bin" \
```

```suggestion
      --prefix PATH : ${lib.makeBinPath [ jre which ]} \
      "${gappsWrapperArgs[@]}"
```
We can use `--prefix PATH` and `lib.makeBinPath` instead. We should also use the flags from `wrapGAppsHook3` here